mirar/pipelines/winter/config/files/avro_schema/winter.alert.prv_candidate.avsc

```diff
@@ -8,13 +8,14 @@
 		{"name": "progname", "type": "string", "doc": "Name of program e.g 2023A000"},
 		{"name": "jd", "type": "double", "doc": "Observation Julian date at start of exposure [days]"},
 		{"name": "fid", "type": "int", "doc": "Filter ID (0=Y; 1=J; 2=H; 3=K)"},
-		{"name": "isdiffpos", "type": ["string", "null"], "doc": "t or 1 => candidate is from positive (sci minus ref) subtraction; f or 0 => candidate is from negative (ref minus sci) subtraction"},
+		{"name": "isdiffpos", "type": "boolean", "doc": "true => candidate is from positive (sci minus ref) subtraction; f or 0 => candidate is from negative (ref minus sci) subtraction"},
 		{"name": "fieldid", "type": "int", "doc": "field ID"},
 		{"name": "ra", "type": "double", "doc": "Right Ascension of candidate; J2000 [deg]"},
 		{"name": "dec", "type": "double", "doc": "Declination of candidate; J2000 [deg]"},
 		{"name": "magpsf", "type": "float", "doc": "Magnitude from PSF-fit photometry [mag]"},
 		{"name": "sigmapsf", "type": "float", "doc": "1-sigma uncertainty in magpsf [mag]"},
 		{"name": "fwhm", "type": ["float", "null"], "doc": "Full Width Half Max assuming a Gaussian core, from SExtractor [pixels]"},
-		{"name": "scorr", "type": ["double", "null"], "doc": "Peak-pixel signal-to-noise ratio in point source matched-filtered detection image"}
+		{"name": "scorr", "type": ["double", "null"], "doc": "Peak-pixel signal-to-noise ratio in point source matched-filtered detection image"},
+		{"name": "deprecated", "type": "boolean", "default": false, "doc": "Whether this candidate is superceded by a reprocessing"}
```

Comment:
This should not be in the schema, because prv_candidates should not include deprecated ones